src/components/Checkbox/Checkbox.js

```diff
@@ -45,16 +52,20 @@ class Checkbox extends Component {
   }
 
   render() {
-    let name = this.props.name || 'checkbox';
-    let hasLabel = !!this.props.label;
-    let classes = classNames('sd-checkbox', 'ui', {fitted: !hasLabel}, this.props.type, 'checkbox');
+    let classes = classNames(
+      'sd-checkbox',
+      'ui',
+      this.props.className,
+      {fitted: !this.props.label},
+      'checkbox'
+    );
+    let checkboxProps = _.clone(this.props);
+    delete checkboxProps.className;
     return (
-      <div className={classes} style={this.props.style} ref='container'>
-        <input defaultChecked={this.props.defaultChecked} type='checkbox' name={name} ref='checkbox' />
+      <div className={classes} ref='container'>
+        <input {...checkboxProps} ref='checkbox' />
```

Comment:
haha this way cleaner than trying to determine whether or not it's a className >.< :1